cmd/ak/cmd/vars/get.go

```diff
@@ -33,12 +33,7 @@ var getCmd = common.StandardCommand(&cobra.Command{
 		ctx, cancel := common.LimitedContext()
 		defer cancel()
 
-		get := vars().Get
-		if reveal {
-			get = vars().Reveal
-		}
-
-		vs, err := get(ctx, id, ks...)
+		vs, err := vars().Get(ctx, id, ks...)
```

Comment:
Please also remove the reveal flag - it's not useful for CLI users, only (maybe) in web UIs.

Reply:
discussed offline: cli is ui, and we don't want to accidently reveal secrets.